src/components/App/App.js

```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import '../../fonts.css';
+import { Container } from '../Container';
+import { Section } from '../Section';
+import { Title } from '../Title';
+import { Card } from '../Card';
+import { array } from '../Carousel/data';
+import { Carousel } from '../Carousel';
+
+export const App = () => {
+  return (
+      <Section className="homes">
+        <Container className="homes-container">
+          <Title title="Homes guests loves" className="homes-title"></Title>
```

Comment:
You may use a closing slash at the end of the JSX tag if it doesn't have children.
```suggestion
          <Title title="Homes guests loves" className="homes-title" />
```